Package.swift

```diff
@@ -5,9 +5,9 @@ import PackageDescription
 let package = Package(
     name: "Generator",
     dependencies: [
-        .package(url: "https://github.com/SDGGiesbrecht/SDGCornerstone", from: Version(2, 6, 0)),
-        .package(url: "https://github.com/SDGGiesbrecht/SDGWeb", from: Version(3, 1, 0)),
-        .package(url: "https://github.com/SDGGiesbrecht/SDGCommandLine", from: Version(1, 2, 0))
+        .package(url: "https://github.com/SDGGiesbrecht/SDGCornerstone", from: Version(4, 2, 0)),
+        .package(url: "https://github.com/SDGGiesbrecht/SDGWeb", from: Version(5, 0, 0)),
+        .package(url: "https://github.com/SDGGiesbrecht/SDGCommandLine", from: Version(1, 2, 4))
```

Comment:
The draft status is because this number needs to be bumped so that the last two are compatible with each other, but the release itself is still pending. I will come back and do that once it is available.